transforms/language/html2parquet/python/README.md

```diff
@@ -155,3 +188,26 @@ Chicago |
 ## Contact Us
 ```
 
+## Usage
+
+### Command-Line Interface (CLI)
+
+Run the transform with the following command:
+
+```
+python ../html2parquet/python/src/html2parquet_transform_python.py \
+  --data_local_config "{'input_folder': '../html2parquet/python/test-data/input', 'output_folder': '../html2parquet/python/test-data/expected'}" \
+  --data_files_to_use '[".html", ".zip"]'
+```
+
+- When invoking the CLI, use the following syntax for these parameters: `--html2parquet_<parameter_name>`. For example: `--html2parquet_output_format='markdown'`.
+
+
+### Sample Notebook
+
+See the [sample notebook](https://github.com/IBM/data-prep-kit/tree/dev/transforms/language/html2parquet/notebooks/html2parquet.ipynb) for an example.
```

Comment:
The link to the notebook did not work for me. I wonder if we should use a relative path such as:
../notebooks/html2parquet.ipynb

Reply:
@touma-I and @sungeunan-ibm The link doesn't work because it is using the dev branch. After merge, it will work. The alternative is to use a relative path, as you suggested.